src/uv_tcp.h

```diff
@@ -13,8 +13,9 @@ struct UvTcp
     struct uv_loop_s *loop;              /* Event loop */
     raft_id id;                          /* ID of this raft server */
     const char *address;                 /* Address of this raft server */
-    unsigned n_listeners;                /* The number of listener sockets (multi IPs) */
-    struct uv_tcp_s* listeners;          /* The array of tcp listen sockets (multi IPs) */
+    unsigned n_listeners; /* The number of listener sockets (multi IPs) */
+    struct uv_tcp_s
+        *listeners; /* The array of tcp listen sockets (multi IPs) */
```

Comment:
I've reworded the comments to fit, it's usually a better solution. If longer comments are needed it's better to put them elsewhere or create a standalone multi-line comment with a dedicated field declaration. This now looks like>

```C
    unsigned n_listeners;                /* Number of listener sockets */
    struct uv_tcp_s *listeners;          /* Listener sockets */
```

(no need to mention that it's an array, because it's our convention already, and multi IP is a feature of the whole thing, not of these particular fields).